src/Core/Services/Implementations/SendService.cs

```diff
@@ -212,7 +212,7 @@ public async Task<(string, bool, bool)> GetSendFileDownloadUrlAsync(Send send, s
             }
 
             send.AccessCount++;
-            await _sendRepository.ReplaceAsync(send);
+            await SaveSendAsync(send);
```

Comment:
Will this have unintended consequences given that this method also calls `ValidateUserCanSaveAsync(send.UserId);` and updates the `RevisionDate` of the Send, vs. just incrementing the count....  I believe the revision date is intended to be an "edit" vs. access, even though you are in theory modifying the underlying record.

Reply:
oop! totally. Good catch. That's a big goof on my part. We just need to trigger the push manually. Thanks, @cscharf